apps/backend/src/datasources/stfc/StfcUserDataSource.ts

```diff
@@ -128,6 +128,11 @@ export class StfcUserDataSource implements UserDataSource {
     StfcUserDataSource.rolesCacheSecondsToLive
   ).enableStatsLogging('uowsRolesCache');
 
+  private uowsRawRolesCache = new Cache<Promise<stfcRole[] | null>>(
```

Comment:
I see where you're coming from with the `raw` names, but I wonder if it might be more intuitive if this new cache was called `uowsRolesCache` or `stfcRolesCache` and the existing one was renamed to `uopRolesCache`?